packages/documentation/src/pages/architecture/site-structure/styling-overview.mdx

```diff
@@ -0,0 +1,74 @@
+# CSS General Info
```

Comment:
I don't know what to do with this. It doesn't really fit under site structure and doesn't really dive into any of the important parts of our styling like the dependency on formation.